cobigen-cli/cli/pom.xml

```diff
@@ -163,20 +163,16 @@
             </arguments>
          </configuration>
       </plugin>
-      <plugin>
-         <artifactId>maven-compiler-plugin</artifactId>
-         <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
-         </configuration>
-      </plugin>
```

Comment:
changing this, would break backward compatibility for java 8 most probably

Reply:
or check wether 

```xml
<properties>
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
  </properties>
```

is set correctly in the properties, which would make this code redundant. Then it's ok to remove it.

Reply:
I have printed all the properties to a console and it seems it is not set. I'm going to revert this change